lumigator/backend/backend/tracking/mlflow.py

```diff
@@ -360,7 +355,7 @@ def delete_workflow(self, workflow_id: str) -> WorkflowResponse:
             name=workflow.data.tags.get("mlflow.runName"),
             description=workflow.data.tags.get("description"),
             model=workflow.data.tags.get("model"),
-            status=WorkflowStatus(workflow.data.tags.get("status")),
+            status=WorkflowStatus(workflow.data.tags.get("status")).value,
```

Comment:
Why are you setting status as the value? WorkflowResponse.status is of WorkflowStatus type, not a str?